sentinel-extension/sentinel-datasource-zookeeper/src/main/java/com/alibaba/csp/sentinel/datasource/zookeeper/ZookeeperDataSource.java

```diff
@@ -39,7 +39,7 @@
     private NodeCacheListener listener;
     private final String path;
 
-    private CuratorFramework zkClient = null;
+    private static CuratorFramework zkClient = null;
```

Comment:
Use upper case for static variable is better for readability.
This way is not thread-safe, use `volatile` and `synchronized` the same as singleton pattern may be better, 
but in most cases it's okay.

Reply:
This is not static final, so I think it should not use upper case.
I already use `synchronized` to ensure thread-safe, it's not necessary to use volatile